src/test/java/testproj/generalSteps.java

```diff
@@ -0,0 +1,157 @@
+    String given_url_global;
+    static String retrieve_properties_variable(String variable_name) {
+
+        String propFileName = "/config/cucumber.properties";
+        Properties prop2 = null;
+        try (InputStream input2 = generalSteps.class.getResourceAsStream(propFileName)) {
+            prop2 = new Properties();
+            prop2.load(input2);
+            return prop2.getProperty(variable_name);
+/*        try (InputStream input = new FileInputStream("src/test/resources/config/cucumber.properties")) {
+
+            prop = new Properties();
+
+            // load a properties file
+            prop.load(input);
+
+            // get the property value and print it out
+*/
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+
+        return prop2.getProperty(variable_name);
+    }
+
+
+    @Before
+    public void createDriver() {
+
+        //create object of chrome options
+        ChromeOptions options = new ChromeOptions();
+
+        //add the headless argument
+        //options.addArguments("headless");
+        options.addArguments("enable-javascript=1");
+        options = new ChromeOptions().setHeadless(true);
+        driver = new ChromeDriver(options);
+    }
+
+    @Given("I start browser")
+    public void i_start_browser() {
+        driver.get("about:blank");
+    }
+
+
+    @When("I open the {string} url")
+    public void i_open_url(String given_url) {
+        System.out.format("Variable name: %s\n", given_url);
+        //given_url_global = retrieve_properties_variable(given_url);
+        given_url_global = given_url;
+        System.out.format("URL to open: %s\n", given_url);
+        driver.get(given_url);
+    }
+
+    @Then("The page loads successfully and HTTP success code is returned")
+    public void i_check_for_success_http_code() throws IOException {
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+
+        HttpURLConnection c=
+                (HttpURLConnection)new
+                        URL(given_url_global)
+                        .openConnection();
+        // set the HEAD request with setRequestMethod
+        c.setRequestMethod("HEAD");
+        // connection started and get response code
+        c.connect();
+        int r = c.getResponseCode();
+        c.disconnect();
+        assertEquals(r, 200);
+        System.out.println("Http response code: " + r);
+    }
+
+    @Then("Title displayed is the {string}")
+    public void check_title(String expected_title) {
+        //assertEquals(retrieve_properties_variable(expected_title), driver.getTitle());
+        assertEquals(expected_title, driver.getTitle());
+    }
+
+    @Then("{string} element is displayed find by {string}")
+    public void check_login_button(String expected_element_id, String item_identifier) {
+        switch (item_identifier) {
+            case "ID" ->
+
+                    //WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
```

Comment:
No commented out code should be commited.

Reply:
Removed.